src/containerapp/azext_containerapp/_help.py

```diff
@@ -1601,3 +1601,50 @@
           az containerapp env telemetry otlp list -n MyContainerappEnvironment -g MyResourceGroup
 """
 
+# DotNet Components Commands
+helps['containerapp env dotnet-component'] = """
+    type: group
+    short-summary: Commands to manage DotNet components within the environment.
+"""
+
+helps['containerapp env dotnet-component list'] = """
+    type: command
+    short-summary: Command to list DotNet components within the environment.
+    examples:
+    - name: List all DotNet components within an environment.
+      text: |
+          az containerapp env dotnet-component list -g MyResourceGroup --environment MyEnvironment
+"""
+
+helps['containerapp env dotnet-component create'] = """
+    type: command
+    short-summary: Command to create DotNet component to enable Aspire Dashboard. Supported DotNet component type is Aspire Dashboard.
```

Comment:
remove aspire dashboard specific message here, you can keep the example as-is since it shows how that type is created

Reply:
Prefer keeping this as-is so customers know this is how they would enable an Aspire Dashboard since the dotnet-component creation command by itself does not provide too much context about how it can be used in relation to enable Aspire.